cypress/e2e/calendar.cy.js

```diff
@@ -66,6 +66,7 @@ describe("Calendar", () => {
 
     cy.get("main").trigger("wheel", { deltaY: -1 });
     tgt.calendar.currentMonthAndYear().contains("January, 2023");
+    cy.wait(200);
```

Comment:
@RascalTwo This PR adds a scroll delay which breaks the test as the test tries to scroll faster than the delay. Is this a good practice with the solution implementation? The scroll delay is 200ms.

Reply:
Yes, this is a valid way to pass the test, additionally, another assertion or test case can be added to try to scroll the page too fast and validate that the month only changed once, but what you have here is satisfactory and additional testing can be done in response to a new issue for it.